### ISSUE TITLE
Remove Fork of NativeEventEmitter.js following 4/29 integration.

### ISSUE THREAD

**Issue by chiaramooney** [microsoft/react-native-windows#7618]
Source as of 4/15 in `Libraries/EventEmitter/NativeEventEmitter.js` contains bug which causes Accessibility Windows page in RNTester to fail to load due to an issue with the addListener method for AppTheme. Fix https://github.com/facebook/react-native/commit/114be1d2170bae2d29da749c07b45acf931e51e2 solves issue and fires a warning instead of a Red Box error. Forked file to create `src/Libraries/EventEmitter/NativeEventEmitter.windows.js` which contains the bug fix. 

TO-DO: Remove forked file following 4/29 integration as bug fix will then be in core.
